CI/setup-windows-sdk.sh

```diff
@@ -172,21 +126,23 @@ while true; do
       break
   fi
 
-  if [ $pacman_attempts -eq 10 ]; then
-    exit 7
-  fi
   pacman_attempts=$((pacman_attempts +1))
 
-  echo "=== Some packages failed to install, waiting and trying again ==="
-  sleep 10
+  if [ ${pacman_attempts} -lt 10 ]; then
+    echo "=== Some packages failed to install, waiting and trying again ==="
+    sleep 10
+  else
+    echo "=== Some packages failed to install after ${pacman_attempts} attempts, giving up ==="
+    exit 7
+  end
```

Comment:
Sorry @vaid2 - I've been hacking away in Lua too much recently (playing After the Plague - tweaking the generic-mapper and writing my own triggers/aliases)
```suggestion
  fi
```